<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Run_Screen</class>
 <widget class="QMainWindow" name="Run_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run_Screen</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>414</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 0);
background-color: rgb(244, 244, 0);</string>
    </property>
    <property name="text">
     <string>ScanDat - Run Automation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Opendir_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Open Directory</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Template</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>938</x>
      <y>130</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Template-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Template-2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Template-3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Template-4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Template-5</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Run_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="Running_status">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>620</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Running Status :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Status_Result">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>620</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="View_results">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>610</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>View_Results</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
